README.md

```diff
@@ -34,14 +34,16 @@ GUARD Tool
 Usage: guard [OPTIONS]
 
 Options:
-  -h,--help              Use the below listed functions.
-                         Warning: Don't try guard on non guardable units
-                         (sys, perv)
-  -c,--create TEXT       Create GUARD record, expects physical path as input.
-  -d,--delete TEXT       Delete GUARD record, expects physical path as input.
-  -l,--list              Listing of GUARDed resources.
-  -r,--clearall          Clears GUARD states for all resources.
-  -v,--version           Version of GUARD tool.
+  -h,--help                Use the below listed functions.
+                           Warning: Don't try guard on non guardable units
+                           (sys, perv)
+  -c,--create TEXT         Create GUARD record, expects physical path as input.
+  -d,--delete TEXT         Delete GUARD record, expects physical path as input.
+  -l,--list                Listing of GUARDed resources.
+  -r,--clearall            Clears GUARD states for all resources.
+  -s,-clearresolvedrecord  Delete GUARD states for resolved resource
```

Comment:
You are deleting only one first record which is resolved so I am not using what the user will do with this

OR

Add description like what the user can expect by this comment.

Reply:
Will change to "Delete GUARD states for oldest resolved resource". 
Mainly for FVT to test the function as they can't access the libguard directly.. So this option is added.

Reply:
Added a note.